pages/01-cartesian-chart/line-dash-styles.page.tsx

```diff
@@ -0,0 +1,97 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { CartesianChart, CartesianChartProps } from "../../lib/components";
+import { dateFormatter } from "../common/formatters";
+import { useChartSettings } from "../common/page-settings";
+import { Page } from "../common/templates";
+
+const domain = [
```

Comment:
Nitpick: this could be generated instead of hard-coded

Reply:
But why would you need to call `pseudoRandom`? This is the x domain, so you could have evenly spaced dates, right?